docs/zh_cn/restful_api.md

```diff
@@ -107,6 +107,14 @@ curl http://{server_name}:{server_port}/v1/embeddings \
   }'
 ```
 
+### webui
+
+也可以直接用 webui 测试使用 restful-api。
+
+```shell
+python -m lmdeploy.serve.gradio.app restful_api_url server_name --restful_api True
```

Comment:
Can `server_name` be put to `restful_api_url`? May provide an example above this command as a comment